apps-devstg/us-east-1/k8s-eks/k8s-workloads/chart-values/demoapps-emojivoto.yaml

```diff
@@ -4,7 +4,7 @@ project: default
 # Source code settings
 source:
   repoURL: https://github.com/binbashar/le-demo-apps.git
-  targetRevision: HEAD
+  targetRevision: head
```

Comment:
It might work both way but I believe the convention has the HEAD in full uppercase.

Reply:
Fixed!